src/main/java/org/usfirst/frc4904/robot/commands/BackLiftSet.java

```diff
@@ -0,0 +1,24 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import edu.wpi.first.wpilibj.command.Command;
+
+public class BackLiftSet extends Command {
+	protected final boolean extended;
+
+	public BackLiftSet(boolean extended) {
+		super(extended ? "BackLiftRetract" : "BackLiftExtend", RobotMap.Component.climber.backLift);
```

Comment:
`BackLiftRetract` and `BackLiftExtend` should be switched here, to match the way you defined them in their individual functions.